pkgs/by-name/de/decent-sampler/package.nix

```diff
@@ -1,32 +1,36 @@
-{ lib
-, stdenv
-, fetchzip
-, fetchurl
-, makeDesktopItem
-, copyDesktopItems
-, buildFHSEnv
-, alsa-lib
-, freetype
-, nghttp2
-, libX11
-, }:
+{
+  lib,
+  stdenv,
+  fetchzip,
+  fetchurl,
+  makeDesktopItem,
+  copyDesktopItems,
+  buildFHSEnv,
+  alsa-lib,
+  freetype,
+  nghttp2,
+  libX11,
+}:
 
 let
   pname = "decent-sampler";
-  version = "1.10.0";
+  version = "1.11.19";
 
   icon = fetchurl {
-    url = "https://archive.org/download/ds-256/DS256.png";
-    hash = "sha256-SV8zY5QJ6uRSrLuGTmT1zwGoIIXCV9GD2ZNiqK+i1Bc=";
+    url = "https://www.decentsamples.com/favicon.ico";
```

Comment:
Isn't this quite low resolution?

Reply:
> Isn't this quite low resolution?

Good point. The favicon is just a redirect to:
https://www.decentsamples.com/wp-content/uploads/2018/09/cropped-Favicon_512x512-32x32.png
which is only 32x32.

I could replace it with e.g.:
https://www.decentsamples.com/wp-content/uploads/2018/09/cropped-Favicon_512x512.png

What do you think?

Reply:
Sounds good, 512px is good enough I'd say.